reference/websocket.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Real-time Websocket
+description: Specifications for interaction with PlaceOS realtime websocket
+---
+
+:::tip
+The real-time WebSocket works in conjunction with the [PlaceOS API](api.md).
+
+The PlaceOS API can list systems and zones that are then used with the WebSocket.
+:::
+
+# Real-time API
+
+PlaceOS exposes the endpoint `wss://<app_domain>/api/engine/v2/systems/control` to allow for real-time communication.
+The endpoint requires a valid API token.
+
+API tokens are sent in the header of the request.
+
+`X-API-Key: <token>`
+
+If you need to ping the channel to keep it alive you can send `"ping"` to which the server will respond `"pong"`.
+
+## Commands
+
+5 commands available for the user to perform over the API.
+These are:
+
+### Bind
+
+`bind` command allows the user to bind to status variable values on a given driver module.
+
+Example of a `bind` command:
+
+```json
```

Comment:
Change these `json` tags to `yaml` and then use yaml comments `#` instead of `//`
it will render the page cleanly